nav.go

```diff
@@ -457,7 +459,28 @@ func (nav *nav) position() {
 	}
 }
 
-func (nav *nav) preview(path string, win *win) {
+func (nav *nav) previewLoop(ui *ui) {
+	var path string
+	for {
+		p, ok := <-nav.previewChan
+		if !ok {
+			return
+		}
+		if len(p) != 0 {
+			win := ui.wins[len(ui.wins)-1]
+			nav.previewGen(p, win)
+			path = p
+		} else if len(gOpts.cleaner) != 0 && len(gOpts.cleaner) != 0 && nav.volatilePreview {
```

Comment:
Is is the same conditional twice?

Reply:
Whoops, I meant to put gOpts.previewer for the first one. Fixed.